src/components/widgets/Plotter.vue

```diff
@@ -0,0 +1,304 @@
+              item-title="name"
+              item-value="id"
+              label="Data Lake variable"
+              hint="Select a variable to be plotted"
+              persistent-hint
+              variant="outlined"
+              density="comfortable"
+              @update:model-value="changeDataLakeVariable"
+            />
+          </div>
+        </v-col>
+      </v-row>
+
+      <!-- Appearance section -->
+      <v-row>
+        <v-col cols="12">
+          <div class="text-subtitle-1 font-weight-medium mb-2">Appearance</div>
+          <div class="ml-2 flex gap-x-8">
+            <v-menu :close-on-content-click="false">
+              <template #activator="{ props: colorPickerActivatorProps }">
+                <div v-bind="colorPickerActivatorProps" class="flex cursor-pointer">
+                  <span class="mt-1">Background color</span>
+                  <div
+                    class="w-[30px] h-[30px] border-2 border-slate-700 rounded-lg cursor-pointer ml-2"
+                    :style="{ backgroundColor: widget.options.backgroundColor }"
+                  ></div>
+                </div>
+              </template>
+              <v-card class="overflow-hidden" :style="interfaceStore.globalGlassMenuStyles">
+                <v-color-picker v-model="widget.options.backgroundColor" label="Background" hide-inputs />
+              </v-card>
+            </v-menu>
+            <v-menu :close-on-content-click="false">
+              <template #activator="{ props: colorPickerActivatorProps }">
+                <div v-bind="colorPickerActivatorProps" class="flex cursor-pointer">
+                  <span class="mt-1">Line color</span>
+                  <div
+                    class="w-[30px] h-[30px] border-2 border-slate-700 rounded-lg cursor-pointer ml-2"
+                    :style="{ backgroundColor: widget.options.lineColor }"
+                  ></div>
+                </div>
+              </template>
+              <v-card class="overflow-hidden" :style="interfaceStore.globalGlassMenuStyles">
+                <v-color-picker v-model="widget.options.lineColor" label="Line" hide-inputs />
+              </v-card>
+            </v-menu>
+            <v-text-field
+              v-model.number="widget.options.lineThickness"
+              type="number"
+              label="Line thickness"
+              variant="outlined"
+              density="compact"
+              :rules="[(v: number) => v > 0 || 'Must be greater than 0']"
+              width="140px"
+              hide-details
+            />
+          </div>
+        </v-col>
+      </v-row>
+
+      <!-- Data points section -->
+      <v-row>
+        <v-col cols="12">
+          <div class="text-subtitle-1 font-weight-medium mb-2">Data Points</div>
+          <div class="ml-2 flex gap-x-8">
+            <v-checkbox v-model="widget.options.limitSamples" label="Limit number of samples" color="primary" />
+            <v-text-field
```

Comment:
Can you make this checkbox white instead of primary? It will be way more visible
![image](https://github.com/user-attachments/assets/2fa83128-8002-4116-b971-03a9b2ba1d53)

Reply:
Done!